packages/@aws-cdk/aws-iam/lib/managed-policy.ts

```diff
@@ -16,14 +16,14 @@ export class AwsManagedPolicy {
   /**
    * The Arn of this managed policy
    */
-  public get policyArn(): string {
+  public policyArn(anchor: cdk.IConstruct): string {
```

Comment:
Maybe you're right.

My motivation for this was that it would be possible to instantiate singleton objects of managed policies (`dynamodb.ReadTablePolicy` or whatever) and they could then be reused in every region.

The alternative is `new dynamodb.ReadTablePolicy(this)`, where you're not actually "creating" a new policy, because it already exists.

But maybe this use case isn't convincing enough.